content/blog/2021-11-17-exp-versioning.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Don't Just Track Your Experiments, Version Them
+date: 2021-11-17
+description: >
+  Versioning ML experiments in DVC helps you log, organize, and reproduce
+  experiments easier than ever.
+descriptionLong: >
+  With DVC experiments, you can take ML experiment tracking to the next level,
+  making it easy and powerful to save, compare, and reproduce experiments at
+  scale.
+picture:
+pictureComment:
+author: dave_berenbaum
+commentsUrl:
+tags:
+  - MLOps
+  - DVC
+  - Experiments
+  - Experiment Tracking
+  - Experiment Versioning
+---
+
+Traditional software version control is not enough for tracking machine learning
+experiments. Over the last few years, experiment tracking has come a long way,
+but there are parallels to problems from earlier generations of version control.
+That's why "experiment versioning" in DVC tracks experiments by building on
+modern version control practices.
+
+# Experiment Tracking Needs
```

Comment:
This paragraph is good but a bit too abstract for the 1st one. I'd suggest to extend it by some examples/pictures or remove/move-for-later.